src/inner_product_proof.rs

```diff
@@ -177,12 +177,21 @@ impl InnerProductProof {
 
     /// Computes three vectors of verification scalars \\([u\_{i}^{2}]\\), \\([u\_{i}^{-2}]\\) and \\([s\_{i}]\\) for combined multiscalar multiplication
     /// in a parent protocol. See [inner product protocol notes](index.html#verification-equation) for details.
+    /// Verifier must provide the input length \\(n\\) explicitly to avoid unbounded allocation within the inner product proof.
```

Comment:
```suggestion
    /// The verifier must provide the input length \\(n\\) explicitly to avoid unbounded allocation within the inner product proof.
```